packages/block-editor/src/components/page-template-picker/templates/blog.native.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const Blog = {
+	// translators: title for "Blog" page template
+	name: __( 'Blog' ),
+	key: 'blog',
+	icon: '📰',
+	content: [
+		{
+			name: 'core/image',
+			attributes: {
+				url:
+					'https://mgblayoutexamples.files.wordpress.com/2020/02/people-woman-coffee-meeting.jpg',
+			},
+		},
+		{
+			name: 'core/spacer',
+			attributes: {
+				height: 24,
+			},
+		},
+		{
+			name: 'core/heading',
+			attributes: {
+				// translators: sample content for "Blog" page template
+				content: __( 'Latest Blog Posts' ),
+				level: 2,
+			},
+		},
+		{
+			name: 'core/latest-posts',
+			attributes: {},
+		},
+		{
+			name: 'core/spacer',
+			attributes: {
+				height: 24,
+			},
+		},
+		{
+			name: 'core/separator',
+			attributes: {},
+		},
+		{
+			name: 'core/spacer',
+			attributes: {
+				height: 24,
+			},
+		},
+		{
+			name: 'core/heading',
+			attributes: {
+				align: 'center',
+				// translators: sample content for "Blog" page template
+				content: __( 'Follow our Blog' ),
+				level: 2,
+			},
+		},
+		{
+			name: 'core/button',
+			attributes: {
+				// translators: sample content for "Blog" page template
+				text: __( 'Subscribe' ),
+				align: 'center',
```

Comment:
I see that button should also be centered, but instead it seems to render left aligned. I didn't see the `align` attribute in the button [block.json](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/block.json) Do you know offhand if this is / will be a supported attribute of the button block?

Reply:
Oh yeah, we don't support that and we are working on `Buttons` block which is a wrapper that has `Button` in it and adds alignment capabilities. I'll remove the attribute.

Reply:
https://github.com/WordPress/gutenberg/pull/21426/commits/9e4ece6fe7d96f9bed6dbf5da1657c1b3306de24